tests/cmdline/commands/test_computer.py

```diff
@@ -515,69 +516,156 @@ def test_show(self):
-        already_existing_filename.mkdir()
-        result = self.cli_runner(computer_export_setup, [sort, comp.label, already_existing_filename], raises=True)
+        already_existing_directory = tmp_path / 'tmp_dir'
+        already_existing_directory.mkdir()
+        result = self.cli_runner(computer_export_setup, [comp.label, already_existing_directory], raises=True)
         assert result.exit_code == ExitCode.CRITICAL
+        assert f'A directory with the name `{already_existing_directory}` already exists.' in result.output
+
+    @pytest.mark.usefixtures('chdir_tmp_path')
+    def test_computer_export_setup_default_filename(self):
+        """Test that default filename is as expected when not specified for `verdi computer export setup`."""
+        comp_label = 'test_computer_export_setup_default'
+        self.comp_builder.label = comp_label
+        # Label needs to be unique during parametrization
+        self.comp_builder.transport = 'core.ssh'
+        comp = self.comp_builder.new()
+        comp.store()
+
+        exported_setup_filename = f'{comp_label}-setup.yml'
 
-    @pytest.mark.parametrize('sort', ['--sort', '--no-sort'])
-    def test_computer_export_config(self, tmp_path, sort):
+        self.cli_runner(computer_export_setup, [comp.label])
+        assert pathlib.Path(exported_setup_filename).is_file()
+
+    @pytest.mark.parametrize('sort_option', ('--sort', '--no-sort'))
+    def test_computer_export_config(self, tmp_path, sort_option, file_regression):
         """Test if 'verdi computer export config' command works"""
-        self.comp_builder.label = 'test_computer_export_config' + sort
+        self.comp_builder.label = f'test_computer_export_config{sort_option}'
         self.comp_builder.transport = 'core.ssh'
         comp = self.comp_builder.new()
         comp.store()
 
         exported_config_filename = tmp_path / 'computer-configure.yml'
+
         # We have not configured the computer yet so it should exit with an critical error
         result = self.cli_runner(computer_export_config, [comp.label, exported_config_filename], raises=True)
         assert result.exit_code == ExitCode.CRITICAL
 
         comp.configure(safe_interval=0.0)
-        result = self.cli_runner(computer_export_config, [comp.label, exported_config_filename])
+        comp.configure(username='aiida')
+
+        result = self.cli_runner(computer_export_config, [comp.label, exported_config_filename, sort_option])
         assert 'Success' in result.output, 'Command should have run successfull.'
         assert (
             str(exported_config_filename) in result.output
         ), 'Filename should be in terminal output but was not found.'
         assert exported_config_filename.exists(), f"'{exported_config_filename}' was not created during export."
+        # file regression test
+        content = exported_config_filename.read_text()
+        file_regression.check(content, extension='.yml')
```

Comment:
This is more of a curiosity than a comment. 

I haven't check how file regression test is done in `aiida` but from my previous experience what I found weird about `file_regression` is it's desire in replacing correct files. 
Suppose the code is doing something funny, and produces wrong results, in this case `file_regression` fails and replaces the correct file with the wrong one.
Then if you run again, it always pass. Because `wrong == wrong`. To be clear running twice always pass.

I discussed this yesterday with Julian, maybe @sphuber have comments how this behavior is being handled in `aiida`

Reply:
That is not how the `file_regression` fixture works. When the reference file does not exist yet, it is created. However, if it already exists and the file generated by the tests is different, it is _not_ replaced but the test simply fails. Only if you run the tests with the `--force-regen` flag, will reference files be updated if they are different

Reply:
That's weird, I never saw it overwriting files. From my experience, when adding a `file_regression`, it fails in the first run, as it doesn't have the file it's supposed to compare against, and creates it, which is also written in the output. Ofc, one could otherwise also add the expected file manually, but I found it quite convenient. Then, once the file is part of the repo, it is used for comparison, but not touched anymore, so the test fails if the implementation changes.